components/org.wso2.carbon.multitenancy.application/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the dependency-reduced-pom.xml which is auto generated. It should not be committed. We have to add the following lines to git.ignore

```
# Ignore dependency-reduced-pom.xml file created by Apache Maven Shade Plugin, see https://maven.apache.org/plugins/maven-shade-plugin/shade-mojo.html
dependency-reduced-pom.xml
```